### PR TITLE
Mark "record structs" and "global usings" as done

### PR DESCRIPTION
**Pull request by jcouv**

### CODE REVIEW

**Review by RikkiGibson — APPROVED** [review]
I wasn't able to find clear indication that we actually merged these in 16.10 and not 16.11, but I think your memory on this is probably better than mine.

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ efforts behind them.
...
-| [Record structs](https://github.com/dotnet/csharplang/issues/4334) | [record-structs](https://github.com/dotnet/roslyn/tree/features/record-structs) | [In Progress](https://github.com/dotnet/roslyn/issues/51199) | [jcouv](https://github.com/jcouv) | [AlekseyTs](https://github.com/AlekseyTs), [RikkiGibson](https://github.com/RikkiGibson) | [jcouv](https://github.com/jcouv) |
-| [Global Using Directive](https://github.com/dotnet/csharplang/issues/3428) | [GlobalUsingDirective](https://github.com/dotnet/roslyn/tree/features/GlobalUsingDirective) | [In Progress](https://github.com/dotnet/roslyn/issues/51307) | [AlekseyTs](https://github.com/AlekseyTs) | [333fred](https://github.com/333fred), [cston](https://github.com/cston) | [AlekseyTs](https://github.com/AlekseyTs) |
+| [Record structs](https://github.com/dotnet/csharplang/issues/4334) | [record-structs](https://github.com/dotnet/roslyn/tree/features/record-structs) | [Merged into 16.10](https://github.com/dotnet/roslyn/issues/51199) | [jcouv](https://github.com/jcouv) | [AlekseyTs](https://github.com/AlekseyTs), [RikkiGibson](https://github.com/RikkiGibson) | [jcouv](https://github.com/jcouv) |
```
Comment (Youssef1313) [inline docs/Language Feature Status.md:13]: Record structs is currently in 16.11, but doesn't seem to be in 16.10

File: docs/Language Feature Status.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ efforts behind them.
...
+| [Global Using Directive](https://github.com/dotnet/csharplang/issues/3428) | [GlobalUsingDirective](https://github.com/dotnet/roslyn/tree/features/GlobalUsingDirective) | [Merged into 16.10](https://github.com/dotnet/roslyn/issues/51307) | [AlekseyTs](https://github.com/AlekseyTs) | [333fred](https://github.com/333fred), [cston](https://github.com/cston) | [AlekseyTs](https://github.com/AlekseyTs) |
```
Comment (Youssef1313) [inline docs/Language Feature Status.md:14]: Global using is also merged to 16.11, but doesn't seem to be in 16.10.

**Comment by jcouv** [comment]
Thanks @Youssef1313. Corrected to 16.11

**Pull request merged**